hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -425,7 +662,10 @@ def _mask_object(object_to_be_masked, topfile):
     try:
         mask = {}
         if topfile is None:
-            topfile = 'salt://hubblestack_nebula_v2/top.mask'
+            # We will maintain backward compatibility by keeping two versions of top files and mask files for now
+            # Once all hubble servers are updated, we can remove old version of top file and mask file
+            # Similar to what we have for nebula and nebula_v2 for older versions and newer versions of profiles
+            topfile = 'salt://hubblestack_nebula_v2/top_v2.mask'
```

Comment:
What backwards-incompatible change happened to the topfile? Can you clarify?

Reply:
Sounds fine. I didn't catch the schema change in my cursory review, I'll look closer.